### DISCUSSION TITLE
spaces in indentation are translated to tabs

### DISCUSSION THREAD

**Discussion by plegrand1** [zim-desktop-wiki/zim-desktop-wiki#2458] [Q&A] [answered]
Hello,
i've got the same proble exposed in this issue : 
https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/859
And i cant understand the solution proposed.
Does someone could help me to solve my problem ?
Thanks.
Pascal

**Comment by plegrand1** [comment, marked as answer]
Hello,
during my research on the internet to try and solve my problem, I discovered that there was a plugin for ZIM "Source_View" (https://zim-wiki.org/manual/Plugins/Source_View.html) which allowed to insert a block code.
This plugin answers my question perfectly.
Pascal

**Reply by brucewestfall** [reply]
Can we please give you a couple hundred thumbs up for actually posting the solution?  It shows how useful these boards are not just for answering a question now, but providing a searchable solution years into the future.

Glad you found the fix!

**Reply by plegrand1** [reply]
Of course you can, but as you can see here: https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/2459, the solution is OK but i've got a litle problem with this plugin !!   ;-)
